usagov/usagov-2021
# USAGOV-1900-wizard-desktop-remove-sidebar: Remove the sidebar for wizards in desktop.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1900

## Description

Update wizard.scss to hide sidebar.

## Type of Changes

- [ X ] Frontend (Twig, Sass, JS)
 ![image](https://github.com/user-attachments/assets/e215e86f-6578-46cb-a170-495bd1207783)
![image](https://github.com/user-attachments/assets/499c2c09-7bf3-4b52-a29b-6f405b19c8a9)

## Testing Instructions


1. Navigate to /admin/structure/menu/manage/left-menu-english.
2. Find and edit the link for "Where to report a scam".
3. Ensure that the link leads to "/scam-reporting" and the "Custom parent" box is checked.
4. Navigate to any wizard page. 
5. The desktop and mobile view of the page and menu matches the screenshots above.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
6. Find the commit of this pull request.
7. Build and deploy the changes.
8. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
